usecases/schema/tenant.go

```diff
@@ -169,45 +172,60 @@ func (h *Handler) DeleteTenants(ctx context.Context, principal *models.Principal
 // GetTenants is used to get tenants of a class.
 //
 // Class must exist and has partitioning enabled
-func (h *Handler) GetTenants(ctx context.Context, principal *models.Principal, class string) ([]*models.Tenant, error) {
+func (h *Handler) GetTenants(ctx context.Context, principal *models.Principal, class string,
```

Comment:
I believe this func is currently unused can you confirm that is correct on both this branch and main and if so, delete it and delete the `getTenants` function in this same file below?